infrastructure/setup-deploy-config.sh

```diff
@@ -13,6 +13,9 @@ echo "Setting up deployment config for $1 - `date --iso-8601=ns`"
 # Set hostname in openhim-console config
 sed -i "s/{{hostname}}/$1/g" /opt/opencrvs/infrastructure/openhim-console-config.deploy.json
 
+# Set ssh user in logrotate.conf
+sed -i -e "s%{{SSH_USER}}%$SSH_USER%" /opt/opencrvs/infrastructure/logrotate.conf
+
```

Comment:
It's best if we keep logrotate and all other server configuration in the server provisioning stage

Reply:
The problem I faced in Niue was that an error is thrown when the logs are rotated and they dont in fact rotate.  This was because they are created with the root user in logrotate.conf and the stack is running under SSH_USER account not root.

Reply:
https://github.com/opencrvs/opencrvs-farajaland/commit/29fe10e7984cdae127357f3d5d55b5364bd58ac1

This commit should now make it so that SSH_USER can access and write to those files